homeassistant/components/sensor/device_trigger.py

```diff
@@ -30,6 +30,7 @@
 
 DEVICE_CLASS_NONE = "none"
 
+CONF_ACCUMULATED_PRECIPITATION = "accumulated_precipitation"
```

Comment:
I suppose this too could use the `homeassistant.const` value.

Reply:
Same here.
All implementations of `device_trigger` currently use their own values in Home Assistant.
That would need to be a separate discussion.